Meadowlark-js/backends/meadowlark-mongodb-backend/docker/scripts/mongo-rs-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+mongo  <<EOF
+var cfg = {
+    "_id": "rs0",
+    "protocolVersion": 1,
+    "version": 1,
+    "members": [
+        {
+            "_id": 0,
+            "host": "mongo1:27017",
+            "priority": 2
```

Comment:
Settings insure mongo1 will be primary, and thus the mongo to connect to for admin user creation